OpenRA.Mods.TS/UtilityCommands/LegacyTilesetImporter.cs

```diff
@@ -57,7 +59,24 @@ public void Run(ModData modData, string[] args)
 				"Cliff" // TS defines this as "Rock"
 			};
 
+			var metadata = new StringBuilder();
+			metadata.AppendLine("General:");
+
+			var name = Path.GetFileNameWithoutExtension(args[1]);
```

Comment:
This now relies on the knowledge of temperat translates to Temperate, but it makes it more flexible I guess.